tests/test_dpa.py

```diff
@@ -4,10 +4,7 @@
 from openshift_resources.data_protection_application import DataProtectionApplication
 
 
-class DynamicClient:
-    pass
-
-
+@pytest.fixture(scope="module")
 def oadp_ocp_client():
```

Comment:
not sure this function is needed.
it's taken care of as part of https://github.com/RedHatQE/openshift-python-wrapper/blob/4d38ca97aeabeed5ce0a521ea083f6e7a8b37c3e/ocp_resources/resource.py#L410